zoninator.php

```diff
@@ -1128,11 +1128,11 @@ function get_zones_for_post( $post_id ) {
 
 	function get_zones( $args = array() ) {
 
-		$args = wp_parse_args( $args, array(
+		$args = apply_filters( 'zoninator_terms_query_args', wp_parse_args( $args, array(
```

Comment:
Since this function already takes `$args` as an argument and we're calling `wp_parse_args` here, I'd be hesitant to filter it like this. I'd be more inclined to change this back as it was, and then instead change [this line](https://github.com/Automattic/zoninator/blob/master/zoninator.php#L262) from:

`$zones = $this->get_zones();`

to

`$zones = $this->get_zones( apply_filters( 'zoninator_admin_page_get_zones_args', array() ) );`

@mjangda thoughts? I'm in support of finally adding such a filter after fielding numerous requests to filter/reorder tabs.